keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -296,35 +664,60 @@ revert to the old behavior where rolling update will proceed one pod at a time.
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 
-We will restore the desired behavior for StatefulSets for which the maxunavailable field wasn't deleted after 
+We will restore the desired behavior for StatefulSets for which the maxUnavailable field wasn't deleted after
```

Comment:
I think this one needs more clarification, I can't fully understand if this is talking about recovering maxUnavailable value? Because if it does, then that's not possible, when we disable the feature we set the field to nil, so you can't get it back .

Reply:
I think the true behavior is 
`If we disable the feature-gate and reenable it again, then the `maxUnavailable` will take effect again,
that's say we'll restart to respect the value of `maxUnavailable` when rolling update.`
Updated.